Mathlib/Analysis/Normed/Field/Lemmas.lean

```diff
@@ -34,50 +31,29 @@ section NormedDivisionRing
 
 variable [NormedDivisionRing α] {a : α}
 
-lemma antilipschitzWith_mul_left {a : α} (ha : a ≠ 0) : AntilipschitzWith (‖a‖₊⁻¹) (a * ·) :=
-  AntilipschitzWith.of_le_mul_dist fun _ _ ↦ by simp [dist_eq_norm, ← _root_.mul_sub, ha]
-
-lemma antilipschitzWith_mul_right {a : α} (ha : a ≠ 0) : AntilipschitzWith (‖a‖₊⁻¹) (· * a) :=
-  AntilipschitzWith.of_le_mul_dist fun _ _ ↦ by
-    simp [dist_eq_norm, ← _root_.sub_mul, ← mul_comm (‖a‖), ha]
-
 /-- Multiplication by a nonzero element `a` on the left
 as a `DilationEquiv` of a normed division ring. -/
 @[simps!]
-def DilationEquiv.mulLeft (a : α) (ha : a ≠ 0) : α ≃ᵈ α where
```

Comment:
Did you intend to make `a` implicit here? I'd argue it should be explicit, to match `Equiv.mulLeft₀ a ha`

Reply:
> arguments which are inferrable from another explicit argument should be implicit.

I claim this only applies to lemmas; with your proposal the goal state shows things like `Equiv.mulLeft₀ _ b = c`, which tells me nothing about what `b` is being multiplied by

Reply:
Good point! I have reverted the change.